hudi-utilities/src/main/java/org/apache/hudi/utilities/sources/helpers/ProtoConversionUtil.java

```diff
@@ -348,17 +348,17 @@ private Object getDefault(Descriptors.FieldDescriptor f) {
         case SFIXED64:
           return 0;
         case UINT64:
-          return "\u0000"; // requires bytes for decimal type
+          return DECIMAL_CONVERSION.toFixed(new BigDecimal(BigInteger.ZERO), fieldSchema, fieldSchema.getLogicalType()).bytes();
```

Comment:
I assume this does not cause backwards compatibility issue.

Reply:
This was broken previously so if you hit this case you'd have a failure

Reply:
Got it sg.